app/assets/javascripts/application.js

```diff
@@ -1,4 +1,5 @@
 //= require helpers
+//= require govuk_publishing_components/components/step-by-step-nav
```

Comment:
See [this link](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/install-and-use.md#4-include-the-assets) for how to do this.